prometheus_pgbouncer_exporter/cli.py

```diff
@@ -11,6 +11,26 @@
 from .collector import PgbouncersMetricsCollector
 
 
+def read_config_file(args):
+    # Read config file
+    config = Config()
+    try:
+        config.read(args.config)
+        logging.getLogger().info("Config file successfully read from {file}".format(file=args.config))
+    except Exception as error:
+        logging.getLogger().fatal("Unable to read config file from {file}".format(file=args.config), extra={"exception": str(error)})
+        sys.exit(1)
```

Comment:
having the function do sys.exit seems wrong... I would just raise..

Reply:
This is still an exit

Reply:
Sorry! I just copied this block over from what they had in their main function, I should have looked at it more carefully.